### ISSUE TITLE
text colour not changing in dark/high contrast theme in ```describe()```

### ISSUE THREAD

**Issue by Hemant2A2** [processing/p5.js-web-editor#2785]
### p5.js version

1.9.0

### What is your operating system?

Mac OS

### Web browser and version

Chrome , Version 120.0.6099.109 (Official Build) (arm64)

### Actual Behavior

when the ```LABEL``` is passed in ```describe()``` the text colour is not changed when switched to dark/high contrast theme.

Light theme:
<img width="1316" alt="Screenshot 2023-12-26 at 4 04 36 PM" src="https://github.com/processing/p5.js-web-editor/assets/137528221/d4c1dc87-4a85-4997-b1a1-0a2fd5bdab13">

Dark theme:
<img width="1382" alt="Screenshot 2023-12-26 at 4 05 03 PM" src="https://github.com/processing/p5.js-web-editor/assets/137528221/9fc2e761-d1d0-43da-8e4c-792006002ffd">

### Expected Behavior

The text colour should toggle as the user switches between light and dark theme

### Steps:
1. Go to https://editor.p5js.org/ and paste the below code in the editor ( or any other code which uses ```describe()```)
### Snippet:
```js

function setup() {
  background('pink');

  // Draw a heart.
  fill('red');
  noStroke();
  circle(67, 67, 20);
  circle(83, 67, 20);
  triangle(91, 73, 75, 95, 59, 73);

  // Add a general description of the canvas
  // and display it for debugging.
  describe('A pink square with a red heart in the bottom-right corner.', LABEL);
}

```

**Comment by sdivyanshu90** [comment]
Hi, @Hemant2A2, thanks for reporting this issue. In my view, this problem is related to `p5.js` rather than `p5.js-web-editor`.

**Comment by lindapaiste** [comment]
We have another issue regarding making the sketch background be white regardless of the theme.  That would also fix this.  See #1852

**Comment by somesh4747** [comment]
@raclim @lindapaiste I have tried to fix the issue #2785, check this #2801

**Comment by Noorain464** [comment]
@raclim @Hemant2A2 I would like to work on this issue
